### PR TITLE
added vpn_type argument on vpn_toggle function

### PR DESCRIPTION
**Pull request by mehdiirh**
added vpn_type argument on vpn_toggle function

### CODE REVIEW

File: huawei_lte_api/api/Vpn.py [hunk 1]
```diff
@@ -19,12 +20,14 @@ def l2tp_settings(self) -> GetResponseType:
...
-    def toggle_status(self, enable: bool = True) -> SetResponseType:
+    def toggle_status(self, vpn_type: VPNType = VPNType.PPTP, enable: bool = True) -> SetResponseType:
         data = {
             "enable": "1" if enable else "0",
         }
+
         return self._session.post_set(
-            "vpn/l2tp_settings",
+            # pptp_settings | l2tp_settings
+            f"vpn/{vpn_type.value}_settings",
```
Comment (Salamek) [inline huawei_lte_api/api/Vpn.py:30]: Please don't use f strings, they are not supported(well) in older versions of python use:
```
'vpn/{vpn_type}_settings.format(vpn_type=vpn_type.value),
```
Thank you

File: huawei_lte_api/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '1.8.1'
+__version__ = '1.8.2'
```
Comment (Salamek) [inline huawei_lte_api/__init__.py:1]: Please do not bump version number, that is done automatically by me on new version release using https://github.com/Salamek/version script that handles version change in all files...

**Review by Salamek — CHANGES_REQUESTED** [review]
Hi, mostly looking good, but needs some small changes. THX!

**Comment by mehdiirh** [comment]
Hi
This lgtm now, what's wrong?

**Comment by Salamek** [comment]
@mehdiirh it is ok now, lint(mypy) for 3.10 went bit "bananas" but don't worry about it, merging...

**Pull request merged**